appveyor.yml

```diff
@@ -79,10 +80,10 @@ install:
 build: false
 
 test_script:
+  # Test package install
   - "python setup.py install"
-  - 'python -c "import ipopt"'
-  - "python test/examplehs071.py"
-  - "python test/exception_handling.py"
-  - "python test/lasso.py"
-  - "conda.exe install --yes scipy"
-  - "python test/rosen.py"
+  - 'python -c "import cyipopt"'
```

Comment:
Same comment on scipy being optional.

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/38b855638492f7f870c55bff52bddba1ba56e46c, https://github.com/matthias-k/cyipopt/pull/75/commits/6d3d4532b10b8af101b321d07228abddc7562a3b, https://github.com/matthias-k/cyipopt/pull/75/commits/69f347ea79f9049fec616a617853ef7c68b4edbd and https://github.com/matthias-k/cyipopt/pull/75/commits/7fca6279bbdc1e2a4380b045e08884abbfda0d6d.

Reply:
See comment on thread two above for explanation.